mmeval/metrics/instance_seg.py

```diff
@@ -0,0 +1,288 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+import warnings
+from copy import deepcopy
+from typing import Dict, List, Optional, Sequence
+
+from mmeval.core.base_metric import BaseMetric
+from mmeval.metrics._vendor.scannet import scannet_eval
+
+
+class InstanceSeg(BaseMetric):
+    """3D instance segmentation evaluation metric.
+
```

Comment:
It is better to add a summary of this Metric. 
i.e. Introduction of ScanNet

Reply:
fixed